apps/react/react-2-native.json

```diff
@@ -43,21 +29,10 @@
       "label": "Template",
       "description": "You can find many grea community templates by searching for <a href='https://www.npmjs.com/search?q=expo-template-*'>expo-template-*</a> on npm.",
       "prompt": true,
-      "prefix": "--template=",
+      "prefix": "-e ",
       "options": [
         { "label": "Blank" },
```

Comment:
Can you please add minimum 5 templates ?

Reply:
added all 3 templates